src/libaktualizr/storage/storage_common_test.cc

```diff
@@ -641,7 +639,7 @@ TEST(StorageImport, ImportData) {
   EXPECT_TRUE(storage->loadTlsCert(&tls_cert));
   EXPECT_TRUE(storage->loadTlsPkey(&tls_pkey));
 
-  // All TLS objects should be updated.
+  // All TLS objects should be updated exept primary keys.
```

Comment:
BTW the keys are for Uptane, not TLS, which is why the previous comment was accurate.